Modify rule S6399: Switch text to progressive education format (APPSEC-425) (#1522)
SonarSource/rspec: rules/S6399/common/fix/introduction.adoc

In most cases, building XML strings with a direct concatenation of user input
is discouraged. While not always possible, using type casting or strong
pattern-based validation can help sanitize tainted inputs.

> However, avoiding handling objects' properties as strings by directly
> constructing Java objects should be the preferred way.

Reviewer: ```suggestion
However, directly constructing Java objects should be preferred
over handling the properties of objects as strings.
```
This is easier to read.
